deploy/olm-catalog/service-binding-operator/0.0.23/service-binding-operator.v0.0.23.clusterserviceversion.yaml

```diff
@@ -41,7 +41,14 @@ metadata:
   namespace: placeholder
 spec:
   apiservicedefinitions: {}
-  customresourcedefinitions: {}
+  customresourcedefinitions:
+      owned:
+      - description: ServiceBinding expresses intent to bind an operator-backed service
```

Comment:
@pedjak do we need to mention `operator-backed service` ? since we also support non-operator backed services as well.

Reply:
You are right, but the source of this line is https://github.com/redhat-developer/service-binding-operator/blob/master/pkg/apis/apps/v1alpha1/servicebinding_types.go#L105 - it would be good revisit the doc/descriptions in a dedicated PR. This PR is just about bringing the things back, i.e. to make CSV valid agin.